filters/builtin/redirect.go

```diff
@@ -149,6 +149,8 @@ func getLocation(ctx filters.FilterContext, location *url.URL, typ redirectType)
 
 	if u.RawQuery == "" {
 		u.RawQuery = r.URL.RawQuery
+	} else {
+		u.RawQuery = r.URL.RawQuery + "&" + u.RawQuery
```

Comment:
Please see the issue comment.
I think we need to have a separate input if you need this behavior, because we do not know how many users could have critical redirects that expect the current behavior.
I think the current behavior is correct.

We could have a 3rd optional parameter to mix the input query and configured query, see also [doc](https://opensource.zalando.com/skipper/reference/filters/#http-redirect).

Reply:
Yeah, I had read the docs, but they say nothing about query (I actually thought when reading the docs they were dropping whatever query was coming from the original request)